plugins/aws/ec2/overutilizedEC2Instance.js

```diff
@@ -0,0 +1,76 @@
+var async = require('async');
+var helpers = require('../../../helpers/aws');
+
+module.exports = {
+    title: 'EC2 CPU Alarm Threshold Exceeded',
+    category: 'EC2',
+    domain: 'Compute',
+    description: 'Identify EC2 instances that have exceeded the alarm threshold for CPU utilization.',
```

Comment:
Change it to something like
```suggestion
    description: 'Ensure EC2 instances do not exceed the alarm threshold for CPU utilization.',
```